### DISCUSSION TITLE
browser shutdown automatically over and over again

### DISCUSSION THREAD

**Discussion by Troylee-rpi** [acheong08/ChatGPT#453] [Q&A]
```
from revChatGPT.ChatGPT import Chatbot
chatbot = Chatbot({
  "session_token": "xxxxx"
}, conversation_id=None, parent_id=None) # You can start a custom conversation

response = chatbot.ask("Prompt", conversation_id=None, parent_id=None) # You can specify custom conversation and parent ids. Otherwise it uses the saved conversation (yes. conversations are automatically saved)

print(response)
```
The above was the code I was running when I got into this situation.

The terminal outputed
```
Spawning` browser...
Browser spawned.
Found Cloudflare Cookie!
Spawning browser...
Browser spawned.
Found Cloudflare Cookie!
```

When I got to "Found Cloudflare Cookie!", the chrome browser popped up  and showed the "connection is secure, proceeding", then the login page appeared. When I was going to press login button, the browser was shut down by itself and the terminal output "Spawning browser...". And it went through this process over and over again.

**Comment by Troylee-rpi** [comment]
It is strange that I used session_token to run the script. It was supposed to skip the login page, right ? Why it popped up ?

**Reply by acheong08** [reply]
It should open up the login page and close it immediately if you don't touch it

**Reply by Troylee-rpi** [reply]
Yes, exactly. But it didn't give any response, and then went to Spawning browser...

**Reply by Troylee-rpi** [reply]
So, what should I do to get any response from chatgpt ?

**Reply by Troylee-rpi** [reply]
> It should open up the login page and close it immediately if you don't touch it

Can you provide me a solution to fix it ?  thanks a lot!

**Reply by kaifeijidezxb** [reply]
Hello, did you solve this problem, I also have this problem, and want to trouble you to see how to solve5555

**Reply by gilgilzuz** [reply]
I got the same problem T.T, whatever i used session token, or email&passwd, this browser pop and shutdown over and over again

**Reply by thurderbird521** [reply]
> I got the same problem T.T, whatever i used session token, or email&passwd, this browser pop and shutdown over and over again

the same to me, is there any solution? it still can not work since i use it
